repository/d.json

```diff
@@ -113,6 +113,18 @@
 				}
 			]
 		},
+		{
+			"name": "Dark Castle Color Scheme",
+			"author": "Scott Grivner",
+			"details": "https://github.com/scottgriv/Dark-Castle-Sublime",
+			"labels": ["theme", "color scheme", "dark theme", "dark castle", "dark"],
```

Comment:
In Sublime Text parlance this is a color scheme and _not_ a theme, so it should also not have that label. Repeating your theme’s name as a label also feels a bit superfluous.

Reply:
Good points. Thank you for your feedback. I'll make the appropriate changes.